src/main/java/com/whalespottingjava/controllers/SightingController.java

```diff
@@ -65,4 +65,11 @@ public String getAllSightings(Model model) {
   @GetMapping("/sightings/approved")
   @ResponseStatus(HttpStatus.I_AM_A_TEAPOT)
   public void getAllApprovedSightings() {}
+
+  @GetMapping("/sightings/pending")
+  @ResponseStatus(HttpStatus.OK)
+  public String getPendingSightings(Model model) {
+    model.addAttribute("sightings", sightingService.getAllPendingSightings());
+    return "sightings_map";
```

Comment:
It is more a question but in the future this method would return a sighting objects or array of object sightings?